olive/passes/onnx/dynamic_to_fixed_shape.py

```diff
@@ -0,0 +1,119 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import logging
+from typing import Any, Callable, Dict, List
+
+from onnxruntime.tools.onnx_model_utils import fix_output_shapes, make_dim_param_fixed, make_input_shape_fixed
```

Comment:
move the import to the run config to decouple the dependency of this pass with ort. From the time being, the setup could be run without ORT installed.